models/load_pearl_of_great_price.py

```diff
@@ -1,12 +1,53 @@
 """Load Pearl of Great Price."""
 
+import re
+
 from bs4 import BeautifulSoup
 from langchain.schema.document import Document
 
 from models.load_utils import clean
 from models.load_utils import to_markdown
 
 
+def remove_links(markdown_content: str) -> str:
```

Comment:
You don't need to worry about links. I will strip out all markdown before indexing.